protocol/message.go

```diff
@@ -130,10 +129,10 @@ func (m *Message) MarshalJSON() ([]byte, error) {
 		ParsedText        json.RawMessage                  `json:"parsedText"`
 		LineCount         int                              `json:"lineCount"`
 		Text              string                           `json:"text"`
-		ChatId            string                           `json:"chatId"`
+		ChatID            string                           `json:"chatId"`
 		LocalChatID       string                           `json:"localChatId"`
 		Clock             uint64                           `json:"clock"`
-		Replace           string                           `json:"replace,omitEmpty"`
+		Replace           string                           `json:"replace"`
```

Comment:
A unit test was failing so I removed it. `omitEmpty` did not work, it is `omitempty`. cc @cammellos

Reply:
good to know, I obviously sprinkled the codebase with the wrong version :)